libdnf/module/module_sack.cpp

```diff
@@ -149,6 +155,28 @@ BaseWeakPtr ModuleSack::get_base() const {
     return p_impl->base;
 }
 
+
+void ModuleSack::Impl::module_defaults_resolve() {
+    module_metadata.resolve_added_metadata();
+    module_defaults = module_metadata.get_default_streams();
```

Comment:
This is an suggestion - what about to call `resolve_added_metadata()` at beginning of `get_default_streams()`?